src/System.Diagnostics.StackTrace/ref/System.Diagnostics.StackTrace.cs

```diff
@@ -172,7 +172,7 @@ public enum SymAddressKind
     }
     public readonly partial struct SymbolToken
     {
-        private readonly int _dummyPrimitive;
+        private readonly object _dummy;
```

Comment:
I don't know why some of these changed from primitive fields to reference type fields.

For instance, this is literally a struct wrapping an int:
https://github.com/dotnet/corefx/blob/16f3eef83d6df4c9dbc0e2992dafbc9553db363d/src/System.Diagnostics.StackTrace/src/System/Diagnostics/SymbolStore/SymbolToken.cs#L9

cc @ericstj

Reply:
Yeah, something looks wrong here...

Reply:
/cc @safern Perhaps this is due to being unable to resolve System.Private.Corelib so GenAPI can't determine that System.Int32 is actually a value type.